assets/template-collection.css

```diff
@@ -283,7 +287,7 @@
   width: 1.6rem;
   height: 1.6rem;
   top: 0.7rem;
-  left: 0.8rem;
+  left: 1rem;
```

Comment:
It also needs to be updated for `input.mobile-facets__checkbox`. When on my iPad, it looks fine on a large screen now but when it goes to the drawer I get the same thing. It's not ideal to tweak it like this as now the checkbox receiving focus isn't quite aligned with the svg but 🤷 

So from what I was testing it looks like left `4.31rem` is what worked to hide the default checkboxes behind the svg on mobile for iOS.

Reply:
Sounds good 👍